server/src/cg/group4/server/database/query/ResetPlayerData.java

```diff
@@ -12,24 +11,25 @@
 public class ResetPlayerData extends Query {
 
     /**
-     *
+     * The player ID to reset.
      */
     protected String cId;
 
+    /**
+     * Creates a new ResetPlayerData Query using the ID of that player.
+     * @param id The ID of the Player.
+     */
     public ResetPlayerData(final String id) {
         cId = id;
     }
 
     @Override
-    public Serializable query(DatabaseConnection databaseConnection) throws SQLException {
-        Statement statement = databaseConnection.query();
-        statement.executeUpdate("DELETE FROM Collectible WHERE GroupId = '" + cId + "'");
-        databaseConnection.commit();
-
-        statement.executeUpdate("DELETE FROM User WHERE Id = '" + cId + "'");
-        databaseConnection.commit();
-
-        statement.close();
+    public Serializable query(final Connection databaseConnection) throws SQLException {
```

Comment:
You no longer remove the player data out of the user table.. Either rename the query or reset the player data in the user table (could also only be the timers)